shipper.py

```diff
@@ -540,14 +601,14 @@ def xstr(s):
 
 # Main
 if __name__ == "__main__":
-    my_version = 7  # update me!
-    my_mod = ''
-    try:
-        my_mod = datetime.fromtimestamp(os.stat(__file__).st_mtime)
-    except OSError as exc:
-        status_note(''.join(('! error: ', exc.args[0], '\n', traceback.format_exc())))
-        sys.exit(1)
-    status_note(''.join(('v', str(my_version), ' - ', str(my_mod))))
+    # my_version = 7  # update me! ## obsolete with microbadger build hash
```

Comment:
if obsolete then I recommend to remove, not comment out. it will still be in git history :-)